.vib/vib-platform-verify.json

```
@@ -34,33 +27,32 @@
       },
       "actions": [
         {
-          "action_id": "trivy",
+          "action_id": "cypress",
           "params": {
-            "threshold": "IGNORE_ALL",
-            "vuln_type": [
-              "OS"
-            ]
+            "timeout_minutes": 50,
```

```suggestion
            "host_aliases" :[
                "carto.vmw"
             ],
            "timeout_minutes": 50,
```
In order to use the new `host_aliases` feature

---

Additionally, we need to remove the hack from the `command.js`, remove `hosts` from `cypress.json` , and ensure we remove any `baseUrl` from `cypress.json`

---

Changed. Thanks !